storage/tokudb/tokudb_sysvars.h

```diff
@@ -93,10 +93,10 @@ extern my_bool      gdb_on_fatal;
 
 extern my_bool         check_jemalloc;
 
-#if TOKUDB_DEBUG
+#if defined(TOKUDB_DEBUG) && TOKUDB_DEBUG
```

Comment:
Now there are two forms of this check: above defined(FOO) and here defined(FOO) && FOO. Should be a single form (IMHO the former one) only